local-docker-compose.yaml

```diff
@@ -122,7 +121,7 @@ services:
         "
         cd "$${ORIG_DIR}"
         # Copy preferences
-        # cp /tmp/matlab.prf /home/muser/.matlab/${MATLAB_VERSION}/matlab.prf
+        cp /tmp/matlab.prf /home/muser/.matlab/${MATLAB_VERSION}/matlab.prf
```

Comment:
Oops. I think for the changes to this file, the prior values seem to be more recent.